src/main/resources/assets/baubles/lang/de_DE.lang

```diff
@@ -8,3 +8,12 @@ button.normal=Normal
 #
 # Keybinding
 keybind.baublesinventory=Baubles-Inventar
+#
+# Description
+name.AMULET=Bauble (Amulet)
```

Comment:
correct. i actually looked it up to be sure about it before but I ended up forgetting to change it.